cypress/e2e/calendar/add-calendar.cy.js

```diff
@@ -0,0 +1,94 @@
+import {
+  TextField, HTML, Calendar, Accordion, MultiSelect, Pane, PaneHeader, MultiColumnList,
+  MultiColumnListCell, MultiColumnListRow, Button, including, Link
+} from '../../../interactors';
+
+import calendarFixtures from '../../fixtures/calendar_e2e_fixtures';
+
+const testServicePoint = calendarFixtures.servicePoint;
+const testCalendar = calendarFixtures.calendar;
+
+
+const toggleCalendarButton = HTML.extend('toggle calendar button').selector('button[id^=datepicker-toggle-calendar-button-dp-]');
+
+describe('Add new calendar for service point', () => {
+  let testCalendarResponse;
+
+  const newCalendarInfo = {
+    name: 'test-calendar-create',
+    startDay: 1,
+    endDay: 2
+  };
+  before(() => {
+    // login and open calendar settings
+    cy.loginAsAdmin();
+
+    // reset db state
+    cy.deleteServicePoint(testServicePoint.id, false);
+
+    // create test service point
+    cy.createServicePoint(testServicePoint, (response) => {
+      testCalendar.assignments = [response.body.id];
+
+      cy.createCalendar(testCalendar, (calResponse) => {
+        testCalendarResponse = calResponse.body;
+      });
+    });
+    cy.openCalendarSettings();
+  });
+
+
+  it('adds new calendar', () => {
```

Comment:
`it('{test case id from test rail} {test case name from test rail} (vega)', { tags: [TestTypes.smoke, devTeams.vega] }, () => {`